src/cloud-api-adaptor/hack/build.sh

```diff
@@ -105,9 +125,15 @@ function build_caa_payload_arch_specific() {
 		--build-arg YQ_VERSION="${YQ_VERSION}" \
 		--build-arg YQ_CHECKSUM="${YQ_CHECKSUM}" \
 		-f cloud-api-adaptor/Dockerfile \
-		${tag_string} \
-		--push \
+		${tag_string_with_tflags} \
+		--load \
 		.
+
+	for tag in ${tag_string}; do
+		echo "Pushing ${tag}"
+		docker push ${tag};
+	done
```

Comment:
Maybe just try to remove this duplicated code by moving to a function?

Reply:
@beraldoleal 
Added a function